actiontext/lib/action_text/plain_text_conversion.rb

```diff
@@ -91,8 +93,9 @@ def remove_trailing_newlines(text)
         text.chomp("")
       end
 
-      def bullet_for_li_node(node, index)
+      def bullet_for_li_node(node)
         if list_node_name_for_li_node(node) == "ol"
+          index = node.parent.children.index(node)
```

Comment:
This will break if an ordered list has text nodes between the items. It's pretty common to have whitespace, like this:

```html
<ol>
<li>foo</li>
<li>bar</li>
</ol>
```

I'm not sure if Action Text ever formats lists this way, but I wanted to mention it.